[ADD] accounting/l10n_es: documentation for new Veri*Factu module (#14086)
odoo/documentation: content/applications/finance/fiscal_localizations/spain.rst

Repeat this operation for all contacts related to the **agriculture** industry.

.. _localizations/spain/veri-factu:

Veri*Factu
==========

.. note::

>    :download:`Download the "declaración responsable"<spain/declaracion_responsable.pdf>`

Reviewer: Maybe I would add a little bit of context or information to know when/why it should be downloaded. What do you think?

<localizations/spain/veri-factu-orders>` to be automatically sent to the tax authorities.

.. _localizations/spain/veri-factu-configuration:

Configuration
-------------

To enable **Veri\*Factu**, follow these steps:

#. Open the Settings app to make sure your company's :guilabel:`Country` and :guilabel:`Tax ID` are
   correctly set in the :ref:`Companies <general/companies/company>` section.
#. :ref:`Install <general/install>` the :guilabel:`Spain - Veri*Factu (l10n_es_edi_verifactu)`
   module.
#. Go to :menuselection:`Accounting --> Configuration --> Settings`, scroll to the
   :guilabel:`Veri\*Factu` section, check the :guilabel:`Enable Veri*Factu` option, and click
   :icon:`oi-arrow-right` :guilabel:`Manage certificates` to add a certificate.
#. In the :guilabel:`Certificates for Veri\*Factu` list view, click :guilabel:`New`.
#. Click :guilabel:`Upload your file`, then select a certificate file and enter the
   :guilabel:`Password` needed to open the certificate (if there is one).

.. note::
   - At least one certificate has to be uploaded.
   - By default Veri*Factu is in testing mode. The data is sent to test servers

>      and is not considered official. When you are ready to send official data to the
>      production servers go to the  :guilabel:`Veri\*Factu` section in the :guilabel:`Settings`
>      and disable :guilabel:`Test Environment`.

Reviewer: We tend to avoid using "you/your" when it's possible.
```suggestion
     and is not considered official. When official data can be sent to the production servers, go to 
     the :guilabel:`Veri\*Factu` section in the :guilabel:`Settings` and disable :guilabel:`Test 
     Environment`.
```

Repeat this operation for all contacts related to the **agriculture** industry.

.. _localizations/spain/veri-factu:

Veri*Factu
==========

.. note::

>    The producer of a Veri*Factu billing system has to self-certify that they comply with the regulations.

Reviewer: Just to be as generic and straightforward as possible for this explanation, as it's not an instruction.
```suggestion
   Producers of Veri*Factu billing systems must self-certify their compliance with the regulations.
```

The :guilabel:`Veri*Factu` checkbox should be checked (since :guilabel:`Enable Veri*Factu` is active in the settings).
Click :guilabel:`Send & Print` again.
This will create an JSON file with information about the invoice and store it as a a Veri*Factu document.
A list of all such documents associated with an invoice can be found in the :guilabel:`Veri*Factu` tab.

If possible the document is send immediately to the AEAT.
This may not happen some times due to waiting time requirements between submissions (imposed by the AEAT).
In that case the document will be send at the next possible time via a scheduled action.

.. note::
   The Veri\*Factu **QR code** is displayed on the invoice PDF.
   By scanning this QR code it can be verified that the invoice is known to the AEAT.

Point of Sale
-------------

Immediatly after order has been paid, the Veri*Factu document will be generated.
Just like for invoices it will be send immediately if possible or at the next possible time via a scheduled action.
If an invoice is created for the order during the payment process the Veri*Factu document will be
generated and send for the invoice instead.

> .. note::
>    The Veri\*Factu **QR code** is displayed on the order receipt.
>    This happens even if an invoice is created for the order.
>    By scanning this QR code it can be verified that the order is known to the AEAT.

Reviewer: ```suggestion
.. note::
   A Veri\*Factu **QR code** appears on the order receipt, even if an invoice is created for the
   order. Scan this code to verify that the invoice has been received and recognized by the
   :abbr:`AEAT (Agencia Estatal de Administración Tributaria)`
```
